### PR TITLE
add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by danilofreire**
Hi Alison!

Thanks very much for your great app! I downloaded it today and I'm very happy with it. To express my gratitude, I'd like to share my Brazilian Portuguese translation. I know there's already a Portuguese translation in the repo (thanks to the translator!), but it doesn't sound very natural to Brazilian Portuguese speakers. So I've decided to add my own, but feel free to discard it if you don't think it's useful. I've translated all the required files plus the motivational quotes. The repository is titled `values-pt-br`. I'm happy to make any changes you think are necessary to make my files fully compatible with the original code. Again, many thanks for writing this lovely little app 😃

### CODE REVIEW

File: StoreListing.md [hunk 1]
```diff
@@ -164,8 +164,30 @@ O Presently permite:
 
 O Presently é 100% gratuito e sem anúncios. Todos os teus registos ficam guardados no teu dispositivo e só tu os podes controlar.
 
-## Arabic
-احتفل بحياتك اليومية مع تطبيق برزنتلي، يومية الامتنان الالكترونية المجانية والخاصة.
```
Comment (alisonthemonster) [inline StoreListing.md:168]: Looks like you accidentally lost some Arabic here as well!

File: app/src/main/res/values-pt-br/array.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <item>Dusk</item>
+        <item>Ivy</item>
+        <item>Wesley</item>
+        <item>Moss</item>
+        <item>Glacier</item>
+        <item>Clean</item>
+        <item>Gelato</item>
+        <item>Waves</item>
+    </string-array>
+</resources>
```
Comment (alisonthemonster) [inline app/src/main/res/values-pt-br/array.xml:17]: No need to translate this file :)

File: app/src/main/res/values-pt-br/attrs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="customAttrs">
+        <attr name="timelineIcon" format="reference" />
+    </declare-styleable>
+</resources>
```
Comment (alisonthemonster) [inline app/src/main/res/values-pt-br/attrs.xml:6]: Same with this one! It can just be deleted.

File: app/src/main/res/values-pt-br/colors.xml [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+    <color name="rosieBackgroundColor">#F1DDD4</color>
+    <color name="rosieMainTextAndButtonColor">#797979</color>
+    <color name="rosieHintQuoteTextColor">#AAA09C</color>
+    <color name="rosieDateTextEntryScreenTextColor">#797979</color>
+    <color name="rosieTimelineColor">#D26635</color>
+    <color name="rosieToolbarColor">#D26635</color>
+    <color name="rosieStatusBarColor">#D26635</color>
+    <color name="rosieToolbarItemColor">#F1DDD4</color>
+
+</resources>
```
Comment (alisonthemonster) [inline app/src/main/res/values-pt-br/colors.xml:248]: This file doesn't need a Portuguese version also.

File: app/src/main/res/values-pt-br/ic_launcher_background.xml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#DBD1C7</color>
+</resources>
```
Comment (alisonthemonster) [inline app/src/main/res/values-pt-br/ic_launcher_background.xml:4]: Same here!

File: app/src/main/res/values-pt-br/styles.xml [hunk 6]
```diff
@@ -0,0 +1,335 @@
...
+
+    <style name="TextAppearance.SearchDate" parent="android:TextAppearance">
+        <item name="fontPath">fonts/value-serif.ttf</item>
+    </style>
+
+    <style name="MilestoneDialog" parent="android:Theme.Dialog">
+        <item name="android:windowNoTitle">true</item>
+        <item name="android:windowIsFloating">true</item>
+    </style>
+</resources>
```
Comment (alisonthemonster) [inline app/src/main/res/values-pt-br/styles.xml:335]: This file also isn't needed!

**Review by alisonthemonster — CHANGES_REQUESTED** [review]
Hello and thank you so much for this contribution! 

A few extra files can be deleted and I just need you to change your folder name from 'values-pt-br` to `values-pt-rBR` and then you're all set!

**Comment by danilofreire** [comment]
Thank you very much for the feedback! Sorry for the mistakes, I will edit the files and create another pull request soon. I will close this one 😄

File: StoreListing.md (hunk 1 above)
Comment (danilofreire) [reply StoreListing.md:168]: Thanks for pointing that out! I've just copied the text from your original file. Should be solved now :)

**Pull request closed without merging**